src/app/contribute/guidelines/component-guidelines/index.html

```diff
@@ -0,0 +1,269 @@
+    </li>
+    <li>
+      Clone your repo locally and run <stache-code>npm install</stache-code> from your local repo's directory to install all required dependencies.
+    </li>
+    <li>
+      <a stacheRouterLink="." fragment="write-visual-regression-tests"></a>Run your first round of visual regression tests</a>. This establishes baseline screenshots of the components in the master repo so that as you make changes and run your visual regression tests, your changes are validated against the state of the repo before the changes.
+    </li>
+  </ol>
+
+  <stache-page-anchor>
+    Write code
+  </stache-page-anchor>
+
+  <ol>
+    <li>
+      Launch a command prompt, <stache-code>cd</stache-code> to the folder where you cloned your branch, and run <stache-code>npm run watch</stache-code>.
+    </li>
+    <li>
+      Launch a second command prompt, <stache-code>cd</stache-code> to the folder where you cloned your branch, and run <stache-code>skyux serve</stache-code>. This launches the SKY UX component demos page where you write the demo components to test your code.
+    </li>
+    <ul>
+      <li>
+        SKY UX demos are authored with <a stacheRouterLink="https://github.com/blackbaud/skyux-cli">SKY UX Builder</a>. If this is your first time using SKY UX Builder, you must install the CLI as a global NPM package before you run <stache-code>skyux serve</stache-code>:
+      </li>
+      <stache-code>$ npm install @blackbaud/skyux-cli -g</stache-code>
+    </ul>
+    <li>
+      When you save code changes, the code compiles, static code analysis is performed, unit tests run, and the page refreshes with the latest changes.
+    </li>
+    <p>
+      Keep in mind that when you create components, you must add public exports through <stache-code>./src/core.ts</stache-code> and <stache-code>./src/demo.ts</stache-code> to allow users to import your component modules. The public exports provide entry points for the bundles that SKY UX deploys.
+    </p>
+    <p>
+      The recommended change detection strategy for SKY UX components is to use <a stacheRouterLink="https://angular.io/api/core/ChangeDetectionStrategy">the <stache-code>OnPush</stache-code> property instead of <stache-code>Default</stache-code></a>.
+    </p>
+  </ol>
+
+  <stache-page-anchor>
+    Write documentation
+  </stache-page-anchor>
+
+  <p>
+    We recommend that you create reference documentation for the <a stacheRouterLink="/components">Components section</a> of the SKY UX website. Component documentation should explain alert-visual features of your components, including all properties and events. In the <a stacheRouterLink="https://github.com/blackbaud/skyux2-docs">SKY UX docs repo</a>, you create subfolders for your components in the <stache-code>src/app/components</stache-code> folder and document them in index.html files. To create action buttons for the Components section landing page, you add entries in the <stache-code>demo-components.service.ts</stache-code> file.
```

Comment:
- Is `alert-visual` rogue here?
- `href` instead of `stacheRouterLink` for the GitHub link.

Reply:
Yup, that was definitely rogue, but it's been updated/removed now.